packages/storybook/stories/CompositeStringUtils.tsx

```diff
@@ -13,3 +13,13 @@ export const CompositeConnectionParamsErrMessage = (errors: string[]): JSX.Eleme
     })}
   </>
 );
+
+export const CreateIdentityLink = (): JSX.Element => {
```

Comment:
This doesn't need to be a function it can just be a constant that is a JSX.Element